### ISSUE TITLE
When  will training code and models be available ?

### ISSUE THREAD

**Issue by luckcodingdog** [Siyu-C/ACAR-Net#2]
Thanks for your ACAR-net paper.
I am looking forward to deeper learning this work.

**Comment by JohannesTK** [comment]
+1

**Comment by yztongzhan** [comment]
+1

**Comment by Vazzz007** [comment]
+1

**Comment by theswerd** [comment]
+1

**Comment by MoonBlvd** [comment]
+1

**Comment by AlexKnightDu** [comment]
already 2021 but

**Comment by suminlee94** [comment]
+1

**Comment by chowkamlee81** [comment]
+1

**Comment by junting** [comment]
Dear All,

Sorry for the delay. We have already released part of our code and we plan to make the rest of the code and models available by next week.

Thanks a lot for your interest in our work! Hope you can find this repo useful!

Junting

**Comment by yaru-zhang** [comment]
Looking forward to the rest of the code and data preparation!!

**Comment by junting** [comment]
Hi @yaru-zhang, 
Data preparation is available now!  The rest will come next.

**Comment by yaru-zhang** [comment]
Hi @junting, there is a problem with data preparation.

I have already known that the detector operates on the center frame. 
I want to know whether the backbone network works on key frames or all extracted frames during training.

Thanks in advance.

**Comment by Siyu-C** [comment]
Hi @yaru-zhang,
The answer is yes. The backbone in our setting operates on multiple frames. You can find more details in [our paper](https://arxiv.org/pdf/2006.07976.pdf).
And we use total batch size 32. You may linearly scale the learning rate if you use a different batch size.

**Comment by yaru-zhang** [comment]
Helpful! Thanks for the prompt reply.

**Comment by yaru-zhang** [comment]
Looking forward to the implementation for ACFB!
